src/components/Profile/AddPets.tsx

```diff
@@ -31,17 +32,19 @@ export default function AddPets({ profileId }: { profileId: string }) {
   const utils = api.useUtils();
   const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
     e.preventDefault();
-    if (!file) {
-      return;
+
+    let address = "";
```

Comment:
If there is no file does the photoUrl in your mutation send an empty string as an address? 
If so, does that remove a photo that's in place?
Not critiquing, just curious as to how this works.

Reply:
Disregard, I see that you handle that on the backend. Nevermind.